.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
```

Comment:
**Suggestion**: Specify a more descriptive name for the job to clearly indicate its purpose, enhancing readability and maintainability. <br> **Code Suggestion**: 
 ```
 - name: Run code review agent on every pull request, respond to user comments
 + name: Automated Code Review on PR and Issue Comment Events
 ```